joint_limits:
  "0":
    min_position: -0.2
    max_position: 0.4
    max_velocity: 1.0
    max_effort: 50.0
  "1":
    min_position: -0.5
    max_position: 0.5
    max_velocity: 1.0
    max_effort: 40.0
  "2":
    min_position: -0.4
    max_position: 0.7
    max_velocity: 1.0
    max_effort: 40.0
  "3":
    min_position: 0.0
    max_position: 1.1
    max_velocity: 1.0
    max_effort: 50.0
  "4":
    min_position: -1.0
    max_position: 1.0
    max_velocity: 1.0
    max_effort: 30.0
  "5":
    min_position: -0.4
    max_position: 0.2
    max_velocity: 1.0
    max_effort: 30.0
  "6":
    min_position: -0.5
    max_position: 0.5
    max_velocity: 1.0
    max_effort: 50.0
  "7":
    min_position: -0.7
    max_position: 0.4
    max_velocity: 1.0
    max_effort: 40.0
  "8":
    min_position: -1.1
    max_position: 0.0
    max_velocity: 2.0
    max_effort: 40.0
  "9":
    min_position: -1.0
    max_position: 1.0
    max_velocity: 1.0
    max_effort: 50.0
  